src/components/VAction/VAction.vue

```diff
@@ -1,18 +1,5 @@
 <script>
-/**
- * @param {{ to: string }} props
- * @param {{ attrs: { href: string } }} data
- * @return { string }
- */
-function getTag({ to }, { attrs }) {
-  if (to) {
-    return 'RouterLink';
-  }
-  if (attrs && attrs.href) {
-    return 'a';
-  }
-  return 'button';
-}
+import { getTag } from '../utils.js';
```

Comment:
This function was identical to the one in `VBtn.vue`, so I moved them to the `utils.js` file and imported into both.

Reply:
Yeah, this is actually fine how it was. VBtn was called VAction for a very short time. VAction is actually deprecated and will be removed eventually, but I leave it there in case SOMEONE stumbled upon it and still has it in their project. I dont want it to break, but eventually it will be removed completely at the official v1 release.